README.SRC

```diff
@@ -92,7 +92,7 @@ To compile the packer sources
 
   Finally type
 
-     make
+     make all
```

Comment:
`all:` is the first target in the Makefile, and most `make` with no explicit target should make the first target.  But OK, this change is accepted.

Reply:
That didn't work for me with GNU Make 4.2.1. AFAIK, with GNU Make you need to explicitly set a default target